packages/web-components/src/slider/slider.spec.md

```diff
@@ -0,0 +1,118 @@
+# Slider
+
+## Component Description
+
+The Slider component allows users to quickly select a value by dragging an icon across a bar. It is often used when setting values with a relaxed precision such as audio volume and screen brightness.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/IBCBJxEbPKS7CvLHG55Wn9/Slider?node-id=513%3A510&t=hCOhOiQJVCVNVVJP-0)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr size: "small" "medium" | "medium"
+- @attr vertical: boolean | false
+- @attr current-value: number
+- @attr min: number
+- @attr max: number
+- @attr step: number | 1
+  - If set, slider appearance has ticks for each step value
+- @attr disabled: boolean | false
+- @attr mode: "single-value" "range" | "single-value"
+
+### Outputs
+
+None
+
+### Events
+
+- @change: Fires whenever the slider's value is changed
+
+### Slots
+
+- default: the primary slot intended for the `input` element
+- thumb: the draggable element of the slider
+- lower-thumb: same as `thumb`, but used in `range` mode
+- upper-thumb: same as `thumb`, but used in `range` mode
```

Comment:
If range doesn't exist in v9 and we aren't picking up now, we should remove these.